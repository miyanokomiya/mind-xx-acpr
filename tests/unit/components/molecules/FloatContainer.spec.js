import { mount } from '@vue/test-utils'
import Target from '@/components/molecules/FloatContainer.vue'

describe('@/components/molecules/FloatContainer.vue', () => {
  describe('snapshot', () => {
    it('default', () => {
      const wrapper = mount(Target, {
        propsData: {
          x: 10,
          y: 20,
        },
        slots: {
          default: '<div>slot</div>',
        },
      })
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
