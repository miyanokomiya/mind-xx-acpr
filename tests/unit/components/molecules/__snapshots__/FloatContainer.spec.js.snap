// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/molecules/FloatContainer.vue snapshot default 1`] = `
<div
  style="position: absolute; left: 0px; top: 0px; transform: translate(10px, 20px);"
>
  <div>
    slot
  </div>
</div>
`;
